samples/Zirku/Zirku.Server/Program.cs

```diff
@@ -142,6 +146,45 @@ async Task CreateApplicationsAsync()
             });
         }
 
+        OpenIddictApplicationDescriptor spaClient = new()
+        {
+            ClientId = "spa",
+            Type = ClientTypes.Public,
+            RedirectUris =
+            {
+                new Uri("http://localhost:5112/index.html"),
+                new Uri("http://localhost:5112/signin-callback.html"),
+                new Uri("http://localhost:5112/signin-silent-callback.html"),
+            },
+            Permissions =
+            {
+                Permissions.Endpoints.Authorization,
+                Permissions.Endpoints.Logout,
+                Permissions.Endpoints.Token,
+                Permissions.GrantTypes.AuthorizationCode,
+                Permissions.GrantTypes.RefreshToken,
+                Permissions.ResponseTypes.Code,
+                Permissions.Scopes.Email,
+                Permissions.Scopes.Profile,
+                Permissions.Scopes.Roles,
+                Permissions.Prefixes.Scope + "api1",
+                Permissions.Prefixes.Scope + "api2"
+            },
+            Requirements =
+            {
+                Requirements.Features.ProofKeyForCodeExchange,
+            },
+        };
+        object existingSpaClient = await manager.FindByClientIdAsync(spaClient.ClientId!);
+        if (existingSpaClient == null)
```

Comment:
It's not a pattern we use anywhere else (not saying it's bad approach, but we should be consistent so let's just use `CreateAsync()` for now).